game.py

```diff
@@ -16,6 +16,9 @@ def __init__(self, bg_image_path, keymap, border_width=16, speed=7):
                                                  (600, info.current_h))
         self.border_width = border_width
         self.speed = speed // (len(keymap))
+        
+        # Keys which control each guy (in order of 'guys' array)
+        self.keymap = [i for row in keymap for i in row]
```

Comment:
How come this has two loops when the old code had just one?

Reply:
I'm using list comprehension for flattening the 2-D array which was taken as an input in the constructor

Reply:
Yes, basically in the loop, row was added while looping through the keymap but now you're also looping through the row and adding each item in row when in your loop you just added the rows.

Why change to doing it that way?

Reply:
Okay.